### DISCUSSION TITLE
@whereNull and @whereNotNull  No directive found!

### DISCUSSION THREAD

**Discussion by saeed-rostami** [nuwave/lighthouse#2554] [Q&A] [answered]
Hello, I'm using version 7 of the lighthouse and I implement @whereNull and @whereNotNull directives but error says "debugMessage": "No directive found for `whereNull`",

this is my code
```
 episodes
    (
        draftEpisodes: Boolean @whereNull(key: "published_at")  
): [PodcastEpisode!]! @paginate
```

**Comment by spawnia** [comment, marked as answer]
First of all, there is no version 7 of Lighthouse. `@whereNull` is included by default, so there should be no issue with that. Did you run `php artisan lighthouse:validate-schema`?

**Reply by saeed-rostami** [reply]
thank u, it solved, it was my version problem
